Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3
+
+RUN mkdir /code
```

Comment:
Нет смысла отдельно делать `mkdir`, WORKDIR создаст директорию автоматически, см. https://docs.docker.com/engine/reference/builder/#workdir:

> The WORKDIR instruction sets the working directory for any RUN, CMD, ENTRYPOINT, COPY and ADD instructions that follow it in the Dockerfile. If the WORKDIR doesn’t exist, it will be created even if it’s not used in any subsequent Dockerfile instruction.